targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L475xG/device/TOOLCHAIN_GCC_ARM/STM32L475XX.ld

```diff
@@ -106,6 +106,16 @@ SECTIONS
         __CRASH_DATA_RAM_END__ = .; /* Define a global symbol at data end */
     } > SRAM1
 
+    .lwip :
+    {
+        . = ALIGN(8);
+        __lwip_bss_start__ = .;
+        *mbed-os/features/lwipstack*.o(.bss*)
+        *mbed-os/features/lwipstack*.o(COMMON)
```

Comment:
I don't think LWIP is a defaut feature on all L475 based boards - will that work fine anyway when there are no LWIP symbols at all (mbed2 application for instance)

Reply:
Should work, I tested this precise use case by disabling lwip. I saw no ill effects. Can't comment for mbed2, though.